src/Valleysoft.DockerCredsProvider/CredsProvider.cs

```diff
@@ -116,4 +117,25 @@ private static async Task<ICredStore> GetCredStoreAsync(string registry, IFileSy
 
         throw new CredsNotFoundException($"No matching auth specified for registry '{registry}' in Docker config.");
     }
+
+    private static string ConvertToHostname(string input)
+    {
+        const string https = "https://";
+        const string http = "http://";
```

Comment:
Please capitalize the member names since these are constants: `Http`, `Https`.

Reply:
done.